ui/src/shared/components/TableGraph.js

```diff
@@ -196,18 +197,27 @@ class TableGraph extends Component {
     const {index} = column
     const {tableOptions: {verticalTimeAxis}} = this.props
     const {timeColumnWidth, processedData} = this.state
+    const columnCount = _.get(processedData, ['0', 'length'], 0)
+    let adjustedColumnSizerWidth = columnSizerWidth
 
     const labels = verticalTimeAxis
       ? _.unzip(processedData)[0]
       : processedData[0]
 
     if (labels.length > 0) {
+      if (columnSizerWidth !== timeColumnWidth) {
+        const difference = columnSizerWidth - timeColumnWidth
+        const increment = Math.ceil(difference / columnCount)
+
+        adjustedColumnSizerWidth = columnSizerWidth + increment
+      }
+
       return verticalTimeAxis && labels[index] === 'time'
```

Comment:
it looks like we are only checking the size of the time column when verticalTimeFormat is true. Can you add some logic for if verticalTimeFormat is false, so we take into account when the times are the top row? Right now they still get truncated.

Reply:
ah good catch, totally forgot about horizontal time